frontend/src/components/ShootDetails/GShootExternalToolsCard.vue

```diff
@@ -57,15 +57,26 @@ const configStore = useConfigStore()
 
 const {
   shootMetadata,
+  shootProjectName,
+  shootDomain,
 } = useShootItem()
 
 const items = computed(() => {
   return configStore.externalTools ?? []
 })
 
+const urlData = computed(() => {
+  return {
+    name: shootMetadata.value.name,
+    namespace: shootMetadata.value.namespace,
```

Comment:
use `shootNamespace` and `shootName` instead of  `shootMetadata`

Reply:
thanks, changed it